src/ssl.c

```diff
@@ -56217,6 +56476,216 @@ int wolfSSL_X509_CA_num(WOLFSSL_X509_STORE* store)
     return cnt_ret;
 }
 
+#ifndef NO_WOLFSSL_STUB
+int wolfSSL_X509_STORE_get_by_subject(WOLFSSL_X509_STORE_CTX* ctx, int idx,
```

Comment:
These functions have X509_STORE in the API but if you look at the parameters, they operate on a WOLFSSL_X509_STORE_CTX. Unless the API definition is wrong, put it back with the other functions that operate on a WOLFSSL_X509_STORE_CTX.

Reply:
This is because OpenSSL has this API defined as:
`# define X509_STORE_get_by_subject X509_STORE_CTX_get_by_subject`
Moved back.

Reply:
Thanks.